docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -126,6 +126,34 @@ docker run -it -p 443:443 \
   docker.elastic.co/package-registry/distribution:{version}
 ----
 
+==== Using custom CA certificates
+
+In scenarios where you are using a custom Certificate Authority (CA), you can see errors like this:
+
+[source,json]
+---
+{"type":"log","@timestamp":"2022-03-02T09:58:36-05:00","tags":["error","plugins","fleet"],"pid":58716,"message":"Error connecting to package registry: request to https://customer.server.name:8443/categories?experimental=true&include_policy_templates=true&kibana.version=7.17.0 failed, reason: self signed certificate in certificate chain"}
+---
```

Comment:
Not sure if we need to mention the errors that may appear here. We don't do this in other cases.

Reply:
I added in case it is search this error for troubleshooting.
It's true that the other sections in this page do not show errors, so it makes sense to remove it.

I guess if we want that kind of information should be added to this other page:
- https://github.com/elastic/observability-docs/blob/main/docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc
- https://www.elastic.co/guide/en/fleet/current/fleet-troubleshooting.html

I've rephrased this page and moved this troubleshooting information to that other page.